go/api/api.pb.go

```diff
@@ -1,15 +1,15 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.26.0
-// 	protoc        v3.15.8
```

Comment:
can you update to v3.15.8 so that the diff is slightly smaller?

Reply:
Fixed! Thanks for the catch